src/main/kotlin/no/nav/familie/ba/mottak/task/OppdaterOgFerdigstillJournalpostTask.kt

```diff
@@ -28,17 +28,15 @@ class OppdaterOgFerdigstillJournalpostTask(private val journalpostClient: Journa
         val journalpost = journalpostClient.hentJournalpost(task.payload)
                                   .takeUnless { it.bruker == null } ?: throw error("Journalpost ${task.payload} mangler bruker")
 
-        if (feature.isEnabled("familie-ba-mottak.journalhendelse.fagsystem.fordeling", true)) {
-            sakClient.hentPågåendeSakStatus(tilPersonIdent(journalpost.bruker!!)).apply {
-                if (harPågåendeSakIInfotrygd) {
-                    log.info("Bruker har sak i Infotrygd. Overlater journalføring til BRUT001 og skipper opprettelse av BehandleSak-" +
-                             "oppgave for journalpost ${journalpost.journalpostId}")
-                    return
-                } else if (!harPågåendeSakIBaSak) {
-                    log.info("Bruker på journalpost ${journalpost.journalpostId} har ikke pågående sak i BA-sak. Skipper derfor " +
-                             "journalføring og opprettelse av BehandleSak-oppgave mot ny løsning i denne omgang.")
-                    return
-                }
+        sakClient.hentPågåendeSakStatus(tilPersonIdent(journalpost.bruker!!)).apply {
+            if (harPågåendeSakIInfotrygd) {
+                log.info("Bruker har sak i Infotrygd. Overlater journalføring til BRUT001 og skipper opprettelse av BehandleSak-" +
```

Comment:
Bare vær obs på at et eller annet tidspunkt så vil vi skru håndtering av journalposter i ruting for BAR